.github/workflows/handle-external-events.yml

```diff
@@ -126,6 +126,20 @@ jobs:
         env:
           VERSION: ${{ github.event.client_payload.data.version }}
 
+  update-playground-image-on-release:
+    needs: build-release-musl-docker-image
+
+    name: Update https://playground.ponylang.io/
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run remote command
+        uses: appleboy/ssh-action@ffff33f8fe0318345a4f00f0e847325954b2a3ed
+        with:
+          host: ${{ secrets.PLAYGROUND_HOST }}
+          username: ${{ secrets.PLAYGROUND_ADMIN_USER }}
+          key: ${{ secrets.PLAYGROUND_KEY }}
```

Comment:
All 3 of these secrets have already been setup on the ponyc repo (they are not organization-wide).

Reply:
Is this the root user?

Reply:
The setup instructions for the playground are at:

https://github.com/ponylang/pony-playground/blob/master/INSTALL.md

Current setup requires a user with superuser access to do this.

We decided when setting up the playground that we didn't care enough about it to try to segment it off. The VM is doing one thing and if it gets compromised, we'll tear it down and build another.

I "key" part of this change is that by automating, then we are giving everyone with commit access to the ponyc repo the ability to (should they want), dump the secrets needed to ssh into the playground machine and do administrative stuff.

Given the above, they could do malicious things. However, having commit access to GH, they could already do malicious things if they chose to.

Reply:
I ask this mostly because `update-playground.bash` should be in the user's home.

Reply:
It is.